src/Onboarding/Welcome.tsx

```diff
@@ -23,42 +24,47 @@ const Welcome: FunctionComponent = () => {
   useStatusBarEffect("dark-content")
 
   return (
-    <View style={style.container}>
-      <TouchableOpacity
-        onPress={() => navigation.navigate(Screens.LanguageSelection)}
-        style={style.languageButtonContainer}
-      >
-        <GlobalText style={style.languageButtonText}>{languageName}</GlobalText>
-      </TouchableOpacity>
-      <View>
-        <Image
-          source={Images.PeopleOnNetworkNodes}
-          style={style.image}
-          accessible
-          accessibilityLabel={t("onboarding.welcome_image_label")}
+    <>
+      <GradientBackground />
```

Comment:
Could we have this component render it's children? or would that mess with the layout?

Reply:
That's what I tried first, and it would be ideal, but it messes up all the tests. For some reason all the `getBy` stuff doesn't work.

Reply:
Can you think of a way around that @aledustet?

Reply:
I'm going to get this merged in for now, but perhaps we can take a look at this once things cool down a bit for you @aledustet?